Common/VR/PPSSPPVR.cpp

```diff
@@ -285,9 +285,19 @@ void UpdateVRProjection(float* projMatrix, float* leftEye, float* rightEye) {
 	VR_TweakProjection(projMatrix, leftEye, VR_PROJECTION_MATRIX_LEFT_EYE);
 	VR_TweakProjection(projMatrix, rightEye, VR_PROJECTION_MATRIX_RIGHT_EYE);
 	VR_TweakMirroring(projMatrix);
+
+	// Set 6DoF scale
+	float scale = pow(fabs(projMatrix[14]), 1.15f);
+	if (PSP_CoreParameter().compat.vrCompat().UnitsPerMeter > 0) {
+		scale = PSP_CoreParameter().compat.vrCompat().UnitsPerMeter;
+		VR_SetConfig(VR_CONFIG_6DOF_PRECISE, true);
+	} else {
+		VR_SetConfig(VR_CONFIG_6DOF_PRECISE, false);
+	}
+	VR_SetConfig(VR_CONFIG_6DOF_SCALE, (int)(scale * 1000000));
 }
 
-void UpdateVRView(float* projMatrix, float* leftEye, float* rightEye) {
-	VR_TweakView(leftEye, projMatrix, VR_VIEW_MATRIX_LEFT_EYE);
-	VR_TweakView(rightEye, projMatrix, VR_VIEW_MATRIX_RIGHT_EYE);
+void UpdateVRView(float* leftEye, float* rightEye) {
```

Comment:
nit: We prefer putting the * against the variable not the type.  Consider: `float *ptrToFloat, notAPointer;`.  It'd be nice if this wasn't how C++ worked, but pretending it works in a more sensible way in our syntax won't make it true.

-[Unknown]